src/AzureIoTHub.Portal/Client/Pages/DeviceModels/LoRaWAN/CreateLoraDeviceModel.razor

```diff
@@ -52,7 +52,7 @@
                         <MudItem md="6" xs=12>
                             <MudGrid>
                                 <MudItem md="6" xs="12">
-                                    <MudNumericField id="@nameof(LoRaDeviceModel.KeepAliveTimeout)" @bind-Value="@LoRaDeviceModel.KeepAliveTimeout" For="@(() => LoRaDeviceModel.KeepAliveTimeout)" Min=1 Label="Device Connection Timeout" Variant="Variant.Outlined"></MudNumericField>
+                                    <MudNumericField id="@nameof(LoRaDeviceModel.KeepAliveTimeout)" @bind-Value="@LoRaDeviceModel.KeepAliveTimeout" For="@(() => LoRaDeviceModel.KeepAliveTimeout)" Min=1 Label="Connection Timeout" Variant="Variant.Outlined"></MudNumericField>
                                 </MudItem>
```

Comment:
I would prefer to modify the the MudItem classes ``md`` ``sm`` and ``xs`` to have a responsive design instead of changing the Label...